### PR TITLE
fix(DIST-304): Rerelease the package to npmjs

### PR DESCRIPTION
**Pull request by mathio**

### CODE REVIEW

File: .releaserc [hunk 1]
```diff
@@ -10,7 +10,7 @@
     [
       "@semantic-release/exec",
       {
-        "successCmd": "npm publish --registry https://registry.npmjs.org"
+        "successCmd": "npm config set @typeform:registry https://registry.npmjs.org/ && npm publish --registry https://registry.npmjs.org"
```
Comment (otorrillas) [inline .releaserc:13]: That should make the trick. Although we probably don't need the last bit anymore:
```suggestion
        "successCmd": "npm config set @typeform:registry https://registry.npmjs.org/ && npm publish"
```

**Comment by travis-tf** [comment]
*[BOT]* Preview available: https://embed.typeform.com/1744a9110e2fcad6914b3d1bd165466e5a57445a/embed.js

**Comment by travis-tf** [comment]
*[BOT]* Preview available: https://embed.typeform.com/0a957266731efebc6d3964bc70aacc0a547b9504/embed.js

**Pull request merged**